src/plugins/intel_gpu/src/runtime/layout.cpp

```diff
@@ -214,6 +214,8 @@ static format to_weights_format(format f, bool is_grouped) {
                 throw std::runtime_error("Invalid conversion of data format to weights format. bfwzyx can't be non-grouped as 4D spatials are not supported");
             return format::goizyx;
         }
+        case format::b_fs_yx_fsv16:
+            return format::o_is_yx_isv16;
```

Comment:
Just want to check: Is the weight tensor of the original target problem not from the constant weight but from the other normal operation? 
Then, I think we should add reorder between the weight tensor and the convolution.
Could you please check the detailed situation?

Reply:
Please refer to this commit: b492b591363fe5493b02656a86ab186c910b00ab

Reply:
Checked offline: the preferred weight format is correctly added by post optimize weight pass